docs/testing-nuget-packages.md

```diff
@@ -0,0 +1,76 @@
+## Testing nuget package for new react-native init templates.
```

Comment:
There's been back and forward discussion about the right thing to do, but some "How to" style docs have also been ending up in the wiki.

Reply:
@asklar has been guiding me to put docs here.
i.e. in the repo if they are for developers working in this repo *on* react-native-windows instead of wiki (whcih I beleive is also for that, and we should probably consolidate.
and the website and docs repo contains for developers *using* react-native-windows

Reply:
@NickGerleman  we had this conversation about a month ago and you and Andrew pushed for putting the docs in the repo. We should pick one option and stick with it.

Reply:
For me, in the physical repo or the wiki of the repo doesn't make too much of a difference (since both are repo local), but yeah, we should have that discussion. I know @kmelmon has been pushing some new stuff to the wiki, but wikis vs docs folder came up this morning again. My own preference is probably wiki over the docs folder but it's not a super strong one.

Reply:
The other interesting case is stuff that's package local. E.g. README's need to go with a package. That seems potentially additive to other docs though.